### PR TITLE
CATROID-1176 Added test for creating project and afterwards checking …

### PR DESCRIPTION
**Pull request by dorianpercic**
…if avaliable on device

*Please enter a short description of your pull request and add a reference to the Jira ticket.*

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [x] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/CreateProjectTest.kt [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        TestUtils.deleteProjects(javaClass.simpleName)
+        PreferenceManager.getDefaultSharedPreferences(applicationContext)
+            .edit()
+            .putInt(AGREED_TO_PRIVACY_POLICY_VERSION, privacyPreferenceSetting)
+            .commit()
+    }
+
+    @Test
+    fun testCheckIfProjectCreated() {
+        var newProjectName = "newProjectTest"
```
Comment (ndrnour) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/CreateProjectTest.kt:102]:
```suggestion
        val newProjectName = "newProjectTest"
```

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/CreateProjectTest.kt [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        onView(withId(R.id.newProjectFloatingActionButton))
+            .check(matches(isDisplayed()))
+            .perform(ViewActions.click())
+        waitFor()
+        val defaultProjectName = ApplicationProvider.getApplicationContext<Context>()
+            .getString(R.string.default_project_name, 1)
+        onView(allOf(withText("$defaultProjectName (1)"), isDisplayed(), instanceOf
+            (EditText::class.java)))
+            .perform(ViewActions.replaceText(newProjectName))
```
Comment (ndrnour) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/CreateProjectTest.kt:111]:
```suggestion
        onView(allOf(withId(R.id.input_edit_text), isDisplayed(), instanceOf(EditText::class.java)))
            .perform(ViewActions.replaceText(newProjectName))
```

**Review by ndrnour — CHANGES_REQUESTED** [review]
please fix the warnings and consider my suggestions

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/CreateProjectTest.kt [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                CATROBAT_TERMS_OF_USE_ACCEPTED
+            ).commit()
+
+        createProject()
+        baseActivityTestRule.launchActivity(null)
+    }
+
+    @After
+    fun tearDown() {
+        TestUtils.deleteProjects(javaClass.simpleName)
```
Comment (ndrnour) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/CreateProjectTest.kt:93]: this line is not needed, the BaseActivityTestRule will take care of that.
```suggestion
        
```

**Comment by dorianpercic** [comment]
> please fix the warnings and consider my suggestions

I committed your suggestions!

**Pull request closed without merging**